All_Relationships/3.model-label-functions.py

```diff
@@ -145,37 +146,27 @@
 # 
 # The following code below trains two different generative models. One model follows the assumption that each label function is independent of each other, while the other model assumes there are dependancies between each function (e.g. $L_{1}$ correlates with $L_{2}$).
 
-# In[12]:
+# In[ ]:
 
 
-def get_columns(lf_hash, lf_name):
-    """
-    This function is designed to extract the column positions of
-    each individual label function given their category (i.e. CbG_DB or DaG_TEXT ...)
-    
-    returns a list of column positions that corresponds to each label function 
-    """
-    return list(
-    map(lambda x: L_train.key_index[x[0]],
-    session.query(L_train.annotation_key_cls.id)
-         .filter(L_train.annotation_key_cls.name.in_(list(lf_hash[lf_name].keys())))
-         .all()
-        )
-    )
+cg_db = get_columns(session, L_train, CG_LFS, "CbG_DB")
+cg_text = get_columns(session, L_train, CG_LFS, "CbG_TEXT")
+dg_text = get_columns(session, L_train, DG_LFS, "DaG_TEXT")
 
 
-# In[13]:
+# In[ ]:
 
 
-cg_db = get_columns(CG_LFS, "CbG_DB")
-cg_text = get_columns(CG_LFS, "CbG_TEXT")
-dg_text = get_columns(DG_LFS, "DaG_TEXT")
+lfs_columns = [
+    cg_text
+]
 
-lfs_columns = [cg_db, cg_text,dg_text, cg_db+cg_text, cg_db+cg_text+dg_text]
-model_names = ["CbG_DB", "CbG_TEXT", "DaG_TEXT", "CbG_ALL", "DaG_CbG_ALL"]
+model_names = [
+    "CbG_TEXT"
+]
 
```

Comment:
Will you be running multiple models at one time (i.e. will the list every have a length > 1)?  Maybe add a comment with the list of possible model choices